common/include/fwd-event.h

```diff
@@ -15,6 +15,7 @@ struct d_change_event;
 struct d_select_event;
 
 enum event_type : unsigned;
+enum class window_event_result : unsigned;
```

Comment:
I tried to declare it as a `uint8_t` but even when I included `pstypes.h`, above or below the `#pragma once`, it still told me that `uint8_t` is an undefined type.